docker/diffusers-flax-cpu/Dockerfile

```diff
@@ -43,6 +43,9 @@ RUN python3 -m pip install --no-cache-dir --upgrade pip uv==0.1.11 && \
         numpy==1.26.4 \
         scipy \
         tensorboard \
-        transformers
+        transformers \
+        hf_transfer
+
+ENV HF_HUB_ENABLE_HF_TRANSFER=1
```

Comment:
I would rather set this in the workflow file that runs CI instead of set in Dockerfile files.

Reply:
That works for me. Let me know if you have any other comments.

Reply:
no , that's all

Reply:
BTW, my comment is not about if it works this way or another way. It's more about the explicitness and to set  environment variables in the place where we actually use it.